### PR TITLE
Adding key attribute to the card component

### PR DESCRIPTION
**Pull request by BSlaven**
I added a key attribute to the card component.

This is from issue:  #11.

An ID is usually used for these situations as they are unique for every element.

Since data we are currently working with here do not have IDs, I used the event's description for the key attribute. 

We can easily change that later on depending of course what will our data look like.  

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/EventCalendar/pull/28"></a>

### CODE REVIEW

**Comment by mohan-murali** [comment]
I don't see any changes. Did you forget to stage the file?

File: src/App.js [hunk 1]
```diff
@@ -38,7 +38,11 @@ function App() {
       </header>
 
       {events.map((event) => (
-        <Card title={event.title} subTitle={event.date.start}>
+        <Card 
+          title={event.title} 
+          subTitle={event.date.start}
+          key={event.description}
```
Comment (mohan-murali) [inline src/App.js]: description might not be unique. You can access the index as the second parameter of the map function and add the key 
```jsx
key={`${index}-${event.description}`}
```

**Comment by mohan-murali** [comment]
Also please remove the package-lock.json file from the commits.

File: src/App.js (hunk 1 above)
Comment (BSlaven) [reply src/App.js]: Is this PR still relevant? I cannot see the card component anywhere now.

**Comment by BSlaven** [comment]
Is it OK to remove package-lock.json file from commits?

File: .gitignore [hunk 2]
```diff
@@ -21,3 +21,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+package-lock.json
```
Comment (eddiejaoude) [inline .gitignore]: this file should not be ignored, people should not commit files they did not make changes to intentionally

File: src/App.js (hunk 1 above)
Comment (eddiejaoude) [reply src/App.js]: The default branch `main` has moved forwards, please update your `main` and merge it into your branch (the PR will update automatically)

**Comment by eddiejaoude** [comment]
I don't see any changes in this PR apart from the lock file?

**Comment by BSlaven** [comment]
I created this PR when there was a Card component generated from an array and as such required a key attribute.
At this moment I think this PR is irrelevant.

**Comment by eddiejaoude** [comment]
ok, I will close and a new one can be created if required

**Pull request closed without merging**